dm/dm/config/subtask_test.go

```diff
@@ -171,6 +171,87 @@ func (t *testConfig) TestSubTaskBlockAllowList(c *C) {
 	c.Assert(cfg.BAList, Equals, filterRules2)
 }
 
+func (t *testConfig) TestSubTaskAdjustLoaderS3Dir(c *C) {
+	cfg := &SubTaskConfig{
+		Name:     "test",
+		SourceID: "source-1",
+		Mode:     ModeAll,
+	}
+
+	// default loader
+	cfg.LoaderConfig = DefaultLoaderConfig()
+	err := cfg.Adjust(false)
+	c.Assert(err, IsNil)
+	c.Assert(cfg.LoaderConfig.Dir, Equals, defaultDir+"."+cfg.Name)
+
+	// file
+	cfg.LoaderConfig = LoaderConfig{
+		PoolSize:   defaultPoolSize,
+		Dir:        "file:///tmp/storage",
```

Comment:
you can check documents or tests to know what format is more widely used to express local files. Please add a test for it

Reply:
ok